projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/lang/zh_cn.json

```
@@ -1466,35 +1466,35 @@
-    "pneumaticcraft.gui.tab.problems.charging_station.no_item": "§f没有可充/放其的物品${br}§0向充气槽位放置气动物品。",
-    "pneumaticcraft.gui.tab.problems.assembly_controller.no_program": "§f没有程序！${br}§0向装配控制器放入一个装配程序。",
-    "pneumaticcraft.gui.tab.problems.assembly_controller.missingMachine": "§f缺失机器！${br}§0并非所选程序所需的每一台机器都存在。向装配线添加一台%s。",
-    "pneumaticcraft.gui.tab.problems.assembly_controller.duplicateMachine": "§f检测到重复机器！${br}§0确保只有一台%s。",
-    "pneumaticcraft.gui.tab.problems.amadron.tooMuchFluid": "§f订购流体量过大！${br}§0一台亚马龙无人机的最大装载流体量为 36 桶。你需要把订单分为多个订单。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughStock": "§f库存不足！${br}§0这已经是最大供货量！",
-    "pneumaticcraft.gui.tab.problems.amadron.tooManyItems": "§f订购物品量过大！${br}§0一台亚马龙无人机的最大装载物品量为 36 组。你需要把订单分为多个订单。",
-    "pneumaticcraft.gui.tab.problems.amadron.outOfStock": "§f库存空！${br}§0稍等，或许供货商会补足库存...",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughItemSpace": "§f目标容器没有足够的空间。${br}§在绑定的物品容器中清出足够的空间。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughFluidSpace": "§f目标容器没有足够的空间。${br}§0确保绑定的容器中没有其余流体，或者绑定更大的流体容器。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughItems": "§f没有可售出的物品！${br}§0订单量已到最大。若需要，向绑定的容器加入更多物品。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughFluid": "§f没有可售出的流体！${br}§0订单量已到最大。若需要，向绑定的容器加入更多流体。",
-    "pneumaticcraft.gui.tab.problems.amadron.noItemProvider": "§f未选择物品存储。${br}§0使用亚马龙平板电脑右击物品存储。",
-    "pneumaticcraft.gui.tab.problems.amadron.noFluidProvider": "§f未选择流体存储。${br}§0使用亚马龙平板电脑右击流体存储。",
-    "pneumaticcraft.gui.tab.problems.airLeak": "§f空气泄漏！${br}§0在输出面连接管道或机器。",
+    "pneumaticcraft.gui.tab.problems.charging_station.no_item": "§f没有可充/放气的物品${br}§0向充气槽位放置气动物品。",
+    "pneumaticcraft.gui.tab.problems.assembly_controller.no_program": "§f没有程序${br}§0向装配控制器放入一个装配程序。",
+    "pneumaticcraft.gui.tab.problems.assembly_controller.missingMachine": "§f缺失机器${br}§0并非所选程序所需的每一台机器都存在。向装配线添加一台%s。",
+    "pneumaticcraft.gui.tab.problems.assembly_controller.duplicateMachine": "§f检测到重复机器${br}§0确保只有一台%s。",
+    "pneumaticcraft.gui.tab.problems.amadron.tooMuchFluid": "§f订购流体量过大${br}§0一台亚马龙无人机的最大装载流体量为 36 桶。你需要把订单分为多个订单。",
+    "pneumaticcraft.gui.tab.problems.amadron.notEnoughStock": "§f库存不足${br}§0这已经是最大供货量！",
+    "pneumaticcraft.gui.tab.problems.amadron.tooManyItems": "§f订购物品量过大！{br}§0一台亚马龙无人机的最大装载物品量为 36 组。你需要把订单分为多个订单。",
+    "pneumaticcraft.gui.tab.problems.amadron.outOfStock": "§f库存空${br}§0稍等，或许供货商会补足库存...",
+    "pneumaticcraft.gui.tab.problems.amadron.notEnoughItemSpace": "§f目标容器没有足够的空间${br}§在绑定的物品容器中清出足够的空间。",
+    "pneumaticcraft.gui.tab.problems.amadron.notEnoughFluidSpace": "§f目标容器没有足够的空间${br}§0确保绑定的容器中没有其余流体，或者绑定更大的流体容器。",
+    "pneumaticcraft.gui.tab.problems.amadron.notEnoughItems": "§f没有可售出的物品${br}§0订单量已到最大。若需要，向绑定的容器加入更多物品。",
+    "pneumaticcraft.gui.tab.problems.amadron.notEnoughFluid": "§f没有可售出的流体${br}§0订单量已到最大。若需要，向绑定的容器加入更多流体。",
+    "pneumaticcraft.gui.tab.problems.amadron.noItemProvider": "§f未选择物品存储${br}§0使用亚马龙平板电脑右击物品存储。",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

不太确定是否可以这样改，但是 inventory 翻译成存储基本没有对的可能。
```suggestion
    "pneumaticcraft.gui.tab.problems.amadron.noItemProvider": "§f未选择物品容器${br}§0手持亚马龙平板电脑右击某个物品容器，即可选中。",
```

---

这里是提供/收取物品/流体的容器，但是存储确实太怪了
这么改了